pkgs/servers/shairport-sync/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, fetchurl, openssl, avahi, alsaLib, libdaemon, autoconf, automake, libtool, popt, unzip, pkgconfig, libconfig, pulseaudio }:
+
+stdenv.mkDerivation rec {
+  version = "2.3.0";
+  name = "shairport-sync-${version}";
+
+  src = fetchurl {
+    url = "https://github.com/mikebrady/shairport-sync/archive/${version}.zip";
+    sha256 = "1kslif2ifrn0frvi39d44wpn53sjahwq0xjc0hd98ycf3xbcgndg";
+  };
+
+  buildInputs = [
+    openssl
+    avahi
+    alsaLib
+    libdaemon
+    autoconf
+    automake
+    pkgconfig
+    libtool
+    popt
+    unzip
+    libconfig
+    pulseaudio
+  ];
+
+  enableParallelBuilding = true;
+
+  preConfigure = "autoreconf -vfi";
+  configureFlags = "--with-alsa --with-avahi --with-ssl=openssl --without-initscript --with-pulseaudio";
+
+  meta = with stdenv.lib; {
+    homepage = https://github.com/mikebrady/shairport-sync;
+    description = " AirTunes emulator. Shairport Sync adds multi-room capability with Audio Synchronisation";
```

Comment:
Apologies for the delay and thanks for the docs. I'll know for next time!

Updated.